src/components/routes/LogViewerRoute.vue

```diff
@@ -196,11 +196,11 @@ export default {
         },
 
         toggleContext(k) {
-            this.$set(this.showContext, k, !this.showContext[k]);
```

Comment:
sorry but this syntax is unreadable to me, can we please stick to boolean assignment?